### PR TITLE
New AreaManager updates

### PR DESCRIPTION
**Pull request by mlubej**
Changes:
- fixed configs
- fixed custom pipelines
- fixed example notebook

### CODE REVIEW

File: GEM/config_files/continuous_monitoring/global_config.json [hunk 1]
```diff
@@ -1,7 +1,7 @@
 {
   "storage": {
     "manager": "eogrow.core.storage.StorageManager",
-    "project_folder": "<your s3 bucket or local path>",
+    "project_folder": "/Users/mlubej/work/projects/sh-project/eo-grow-examples/GEM/project",
```
Comment (zigaLuksic) [inline GEM/config_files/continuous_monitoring/global_config.json]: revert back please

Comment (mlubej) [reply GEM/config_files/continuous_monitoring/global_config.json]: Oopsie. Reverted.

**Comment by zigaLuksic** [comment]
should we do a version restriction in the requirements as well?

**Comment by mlubej** [comment]
> should we do a version restriction in the requirements as well?

Did you mena eo-grow or some specific (which?) packages? or simply everything?

**Comment by zigaLuksic** [comment]
> > should we do a version restriction in the requirements as well?
> 
> Did you mena eo-grow or some specific (which?) packages? or simply everything?

eo-grow to 1.4

**Comment by mlubej** [comment]
> > > should we do a version restriction in the requirements as well?
> > 
> > 
> > Did you mena eo-grow or some specific (which?) packages? or simply everything?
> 
> eo-grow to 1.4

From the examples POV I think it makes sense, yes.

**Comment by mlubej** [comment]
> > > should we do a version restriction in the requirements as well?
> > 
> > 
> > Did you mena eo-grow or some specific (which?) packages? or simply everything?
> 
> eo-grow to 1.4

version fixed to `eo-grow==1.4.0`. let me know if you wanted to have a lower bound instead.

**Pull request merged**